projectq/backends/_circuits/_to_latex.py

```diff
@@ -17,7 +17,7 @@
                           Measure, SqrtSwap, Swap, X, Z)
 
 
-def to_latex(circuit):
+def to_latex(circuit, paler_order = None, paler_one_gate_at_a_time = False):
```

Comment:
Same thing about `paler` which is not clear to me...
And please document the new argument in the function docstring.

Reply:
new PR addresses this comment